docs/source/euler_2d_riemann.rst

```diff
@@ -86,11 +104,11 @@ C++ synopsis
 
      const auto probId = pda::Euler2d::Riemann;
      const auto scheme = pda::InviscidFluxReconstruction::FirstOrder; //or Weno3, Weno5
-     auto problem      = pda::create_problem_eigen(meshObj, probId, scheme [, icId]);
+     auto problem      = pda::create_problem_eigen(meshObj, probId, scheme [, icId, userParams]);
```

Comment:
while here you say that one can do: 
```
pda::create_problem_eigen(meshObj, probId, scheme [, icId, userParams]);
```
in the actual impl this api does not exist (yet). see https://github.com/Pressio/pressio-demoapps/pull/157/files#diff-aa85766a11e06f2569a7753fd2871160869665a71c816435754099cc054ec53b

Reply:
Whoops yeah. Though I feel like providing the overload with the custom BCs is kind of opening another can of worms, having to explain the BC functors. A little difficult to show that concisely, since the BC functors need to be user-supplied.

Reply:
this overload does not have custom BCs, just the parametrization . i added it